src/olympia/migrations/1018-drop-l10n-from-ratings-body.sql

```diff
@@ -0,0 +1,14 @@
+/* Add the new column.*/
+ALTER TABLE `reviews`
+    ADD COLUMN `text_body` longtext NULL;
```

Comment:
Why not simply name this column `body`?

Reply:
The existing column (the fk to `translations`) is already called `body` and we can't rename that without everything blowing up during the deploy.  We could name the field in django something different - that would be a lot of renaming though to something less snappy (I took the lazy choice).

Reply:
Ah, mind adding this as a comment? I'm perfectly fine with that lazy choice then :laughing: